api/proto/info/info.proto

```diff
@@ -405,6 +405,22 @@ enum ZDeviceState {
     ZDEVICE_STATE_BOOTING = 5;
 }
 
+enum StorageStatus {
+  STORAGE_STATUS_UNSPECIFIED = 0;
+  STORAGE_STATUS_ONLINE = 1;   // The device or virtual device is in normal working order.
+  STORAGE_STATUS_DEGRADED = 2; // The virtual device has experienced a failure but can still function.
+  STORAGE_STATUS_FAULTED = 3;  // The device or virtual device is completely inaccessible.
+  STORAGE_STATUS_OFFLINE = 4;  // The device has been explicitly taken offline by the administrator.
+  STORAGE_STATUS_UNAVAIL = 5;  // The device or virtual device cannot be opened. In some cases, pools with UNAVAIL devices appear in DEGRADED mode.
+  STORAGE_STATUS_REMOVED = 6;  // The device was physically removed while the system was running.
+  STORAGE_STATUS_SUSPENDED = 7; // A pool that is waiting for device connectivity to be restored.
+}
+
+message StorageDiskState {
+  string disk_name = 1;                // for example /dev/sda
```

Comment:
I think we should also have a logical name and a serial here to make sure the zmsg and UI work is prepared to handle the different way to identify the disks the same way for info and for config. https://github.com/lf-edge/eve/pull/2531 has a disk_description and maybe that should be added to evecommon so we can share it between info and config subdirs.

Reply:
Note that Petr has updated #2531 so you can import the disk description from evecommon.